pyproject.toml

```diff
@@ -106,7 +107,7 @@ qed = ["fiatlux"]
 nolha = ["pdfflow", "lhapdf-management"]
 
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
```

Comment:
Why did you disable versioning?

(it needs to be reenabled, but i'd like to know the exact problem that lead to beeding to disable it)

Reply:
I think this happened because I built NNPDF locally. I reverted it.

Reply:
But in principle you should be able to install with `pip install  -e .`  (or without `-e`) and it should not modify this file.

Reply:
Strange. It did happen in my case...

Reply:
Is your environment managed by poetry?